release_checklist.sh

```diff
@@ -0,0 +1,254 @@
+#!/bin/bash
+set -euo pipefail
+
+script_path="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+
+source ./release_utils.sh
+
+command -v gh >/dev/null || abort "Error: The Github CLI must be installed."
+
+release_type=""
+release_date=""
+gb_mobile_version=""
+main_apps_version=""
+issue_note=""
+issue_number=""
+include_aztec_steps=""
+include_incoming_changes=""
+debug_template=""
+auto_confirm=""
+
+while getopts "t:v:d:g:m:i:axhuy" opt; do
+  case ${opt} in
+    h )
+      echo "options:"
+      echo "   -t Release type. accepts scheduled|beta|hotfix"
+      echo "   -v Gutenberg Mobile release version"
+      echo "   -m Mobile app version"
+      echo "   -d Release date in YYYY-MM-DD format. Only used to generate scheduled release checklists"
+      echo "   -g Relative path to guteberg mobile"
+      echo "   -n Issue note"
```

Comment:
I tried to use this option but it fails when introducing text, not sure how this text should be provided to the command 🤔 .

**Examples:**
Command: `/release_checklist.sh -n some text`
Result:
```
./release_checklist.sh: illegal option -- n
./release_checklist.sh: line 72: OPTARG: unbound variable
```


Command: `./release_checklist.sh -n "some text"`
Result: 
```
./release_checklist.sh: illegal option -- n
./release_checklist.sh: line 72: OPTARG: unbound variable